infra-as-code/bicep/modules/hubNetworking/README.md

```diff
@@ -134,13 +149,26 @@ $ConnectivitySubscriptionId = "[your platform connectivity subscription ID]"
 
 Select-AzSubscription -SubscriptionId $ConnectivitySubscriptionId
 
-New-AzResourceGroup -Name 'Hub_Networking_POC' `
+# Set Platform management subscripion ID as the the current subscription
+$ManagementSubscriptionId = "[your platform management subscription ID]"
+
+# Set the top level MG Prefix in accordance to your environment. This example assumes default 'alz'.
+$TopLevelMGPrefix = "alz"
+
+$ResourceGroupName = "rg-$TopLevelMGPrefix-hub-networking-001"
```

Comment:
@jtracey93 is this referring to why CLI having all the parameters and values in the final line?  e.g. `az deployment group create --name X --location Y --template-file Z` in comparison to using `@inputObject` for PowerShell?

It seems like tables can't be used for parameters in CLI otherwise it would match PowerShell standards.

If I'm missing something please let me know!

Reply:
Call to clarify, this one has been amended